platform/resources/runJobOnWorker.groovy

```diff
@@ -0,0 +1,7 @@
+import hudson.model.*
+
+JOB_TO_RUN = build.environment.get('JOB_TO_RUN')
+ghprbActualCommit = build.environment.get('ghprbActualCommit')
+WORKER_LABEL = build.environment.get('WORKER_LABEL')
+
+build(JOB_TO_RUN, ghprbActualCommit: ghprbActualCommit, WORKER_LABEL: WORKER_LABEL)
```

Comment:
Ah, thank you. Could you please add a comment above this line with that first link? That would be super helpful.